Subject: Fix missing main category from desktop file
From: Evangelos Ribeiro Tzaras <devrtz-debian@fortysixandtwo.eu>
--- a/data/sm.puri.Chatty-daemon.desktop.in
+++ b/data/sm.puri.Chatty-daemon.desktop.in
@@ -6,7 +6,7 @@
 Keywords=XMPP;SMS;chat;jabber;messaging;modem
 Terminal=false
 Type=Application
-Categories=GNOME;GTK;Chat;InstantMessaging
+Categories=Network;GNOME;GTK;Chat;InstantMessaging
 StartupNotify=true
 NoDisplay=true
 X-GNOME-AutoRestart=true
--- a/data/sm.puri.Chatty.desktop.in
+++ b/data/sm.puri.Chatty.desktop.in
@@ -6,6 +6,6 @@
 Keywords=XMPP;SMS;chat;jabber;messaging;modem
 Terminal=false
 Type=Application
-Categories=GNOME;GTK;Chat;InstantMessaging
+Categories=Network;GNOME;GTK;Chat;InstantMessaging
 StartupNotify=true
 MimeType=x-scheme-handler/sms
